codeforboston/maple
# LinkedIn url update

# Summary

Closes #1285. Removes the prefix for the link to linkedIn. Still not the perfect solution as users will need to enter their whole link otherwise it won't work, but since there are different routes for individuals, companies, schools, etc, I don't know that there is a great alternative.

Also fixed some formatting issues for the edit input fields for social links. Open to feedback if others feel this could be dealt with more elegantly. The Input component has flex and centering formatting, which is causing this to look awkward in prod. Rather than adjusting that and impacting other areas of the site, I just used a module here.

Lastly, added a couple of translations that were missing on the edit profile page.

# Checklist

- [ X] On the frontend, I've made my strings translate-able.
- [ N/A] If I've added shared components, I've added a storybook story.
- [X ] I've made pages responsive and look good on mobile.

# Screenshots

Prod LinkedIn Link:
![prodLinkedIn](https://github.com/codeforboston/maple/assets/97904016/ad940569-2c96-4033-a4e1-a5cf2422bd77)

Updated LinkedIn Link:
![updatedLinkedIn](https://github.com/codeforboston/maple/assets/97904016/a1cd6783-4a86-4f98-8a2d-64b00ef34120)

Prod Social Inputs:
![prodSocialInputs](https://github.com/codeforboston/maple/assets/97904016/0bdefbb8-8ade-44c4-bd9b-ba43ab6b14ed)

Updated Social Inputs:
![updatedSocialInputs](https://github.com/codeforboston/maple/assets/97904016/36ed773e-dd3f-4da3-b1e3-ba99cd7ad2da)


# Known issues

As it stands, people could theoretically add any link as their linkedIn.

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Go to edit profile.
2. Add the Url to your linkedIn. Save
3. Return to your profile and hover the linkedIn icon. See the path.
